src/clientversion.h

```diff
@@ -39,7 +40,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers and The Zcash developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers and The Zcash developers and The Hush developers"
```

Comment:
The Bitcoin Core Developers, The Zcash developers and, The Hush developers" ??

Reply:
I am not super into grammar wars, I just wanted to update the copyright line 👼  Whatever @radix42 wants the string to be, that is what it will be